plutus-core/generators/Language/PlutusCore/Generators/NEAT/Type.hs

```diff
@@ -130,6 +134,49 @@ instance Enumerable tyname => Enumerable (Neutral (TypeG tyname)) where
 
 -- ** Enumerating terms
 
+-- Word8 is enumerable so we get an enumerable instance via pack
+instance Enumerable ByteString where
+  enumerate = share $ fmap pack access
+
+data TermConstantG = TmIntegerG Integer
+                   | TmByteStringG ByteString
+                   | TmStringG String
+                   | TmBoolG Bool
+                   | TmUnitG ()
+                   | TmCharG Char
+                   deriving (Show, Eq)
+
+deriveEnumerable ''TermConstantG
+
+data TermBuiltinG = AddIntegerG
```

Comment:
Maybe reuse `DefaultFun` instead? Possibly with a `newtype` wrapper if you need to provide instances for `TermBuiltinG` that override existing ones.